apis/projectcontour/v1/httpproxy.go

```diff
@@ -297,6 +297,7 @@ type TCPHealthCheckPolicy struct {
 // TimeoutPolicy durations are expressed in the Go [Duration format](https://godoc.org/time#ParseDuration).
 // Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 // The string "infinity" is also a valid input and specifies no timeout.
+// A value of "0s" will be treated as if the field were not set, i.e. by using Envoy's default behavior.
```

Comment:
There's 2 comments on the struct fields about the timeout ... are they still accurate?

Reply:
I added some more clarity there around default behavior.